chemicaltoolbox/rdkit/sdf_to_tab.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+import argparse
+import pandas as pd
+from rdkit import Chem
+
+def sdf_to_tab(vars):
+    mols = Chem.SDMolSupplier(vars.inp, sanitize=False)
+    df = pd.DataFrame()  # for output
+
+    for mol in mols:
+        if mol != None:
+            d = mol.GetPropsAsDict()
+            # filter dict for desired props
+            if vars.props == '':  # none specified, return all
+                d = {prop: val for (prop, val) in d.items() if not any(x in str(val) for x in ['\n', '\t'])}  # remove items containing newlines or tabs
+            else:
+                d = {prop: val for (prop, val) in d.items() if prop in vars.props.replace(' ', '').split(',')}  # remove items not requested via CLI
+            d['Name'] = mol.GetProp('_Name')
+        else:
+            d = {'Name': 'ERROR'}
```

Comment:
Is this needed? You can not even track this down to the molecule in question. Should we error out in this case?

Reply:
At least this way there's a record that a molecule was skipped, this is a good thing right?

Reply:
but in the result file? Maybe on the stdout .... 3 molecules skipped or something like this.